be/src/olap/push_handler.cpp

```diff
@@ -829,8 +829,8 @@ Status PushBrokerReader::init(const Schema* schema, const TBrokerScanRange& t_sc
     }
     _runtime_profile = _runtime_state->runtime_profile();
     _runtime_profile->set_name("PushBrokerReader");
-    _mem_pool.reset(new MemPool(_runtime_state->scanner_mem_tracker().get()));
```

Comment:
Using load memtracker here?

Reply:
It has been recorded in batch load thread mem tracker.
![image](https://user-images.githubusercontent.com/13197424/211698626-8a6b9a3e-c2b3-466d-8df3-c40688ec3ed7.png)